src/components/OrganizationMarker.js

```diff
@@ -20,34 +21,51 @@ export class OrganizationMarker extends Component {
   }
 
 
-  handleClick = (e) => {
 
-    this.props.setOpenMarker(this.props.id)
-    this.props.handleClick(this.props.organization.id)
+
+  scrollToElement = (e) => {
+    this.props.setOpenMarker(this.props.orgIndexes[0])
+    if(this.props.orgIndexes.length == 1){
+      this.props.scrollToElement(this.props.orgIndexes[0])
+    }
+  }
+
+  handleClickOfInfoWindow = (e) => {
+    console.log(e.currentTarget.id)
+    this.props.scrollToElement(e.currentTarget.id)
   }
 
   handleClose = () => {
     this.setState({ open: false });
   }
 
   render() {
-    const { organization } = this.props;
+
+    if(this.orgsArray.length == 0){
+      for(var orgIndex of this.props.orgIndexes){
+        this.orgsArray.push([this.props.organizations[orgIndex], orgIndex])
```

Comment:
I think this logic could live in either the Map component around line 42 in the following forEach         `Object.entries(props.locationAddressHashTable).forEach( ([hashIndex,  orgRefs]) => {`
 or here in the OrganizationCard component. I'm curious about how you made the choice?